### DISCUSSION TITLE
The support for this repo / app seems to be so poor

### DISCUSSION THREAD

**Discussion by sayo96** [noah-nuebling/mac-mouse-fix#1108] [Other]
Can someone suggest me a good alternative to this?

**Comment by noah-nuebling** [comment]
Yeah sorry my support really sucks, especially over the last few months. 

I'm working hard on an update, but I sort of get tunnel-visioned with that and ignore everything else which is probably not ideal.

If you want to switch, I can recommend [SteerMouse](https://plentycom.jp/en/steermouse/), [MOS](https://mos.caldis.me/) or [LinearMouse](https://linearmouse.app/).

**SteerMouse** feels a bit old and is a bit expensive but it's very solid software with tons of features and support for different devices that other mouse apps don't support as well.

**MOS** focuses on smooth scrolling. But it's very good at that. It has lots of customization options and a nice UI.

**LinearMouse** is completely free and open source, and it has a great feature set. It doesn't have smooth scrolling or the cool trackpad-gesture simulation of Mac Mouse Fix, but it has really solid app-specific-settings, lots of button-remapping features, and more options for adjusting scrolling-speed and pointer-speed than MMF. I also think the UI is also very competently designed. I don't know how the support is, but it's hard to be worse than mine. Overall I think it's the best alternative to MMF for most users.

---

I also wanna bring all this great stuff to MMF at some point, but this stuff always takes me a long time. And with the support situation I also can't promise that it'll improve anytime soon. 

I hope that helps a bit! If there's something specific that I could do to improve your experience with the app, please also let me know.
